client/src/components/CFooter.style.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+export const Footer = styled.footer`
+  background-color: var(--white);
+  display: flex;
+  flex-direction: column;
+  height: 6rem;
+`;
+export const FooterWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
Great with the flex! If you add ```justify-content: space-between``` here, you won't have to use margins on the links to adjust where they are